boilerplates/custom-taxonomy.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Custom Taxonomy example file
+ *
+ * @package WordPress
+ * @subpackage braces
+ * @author Oomph, Inc.
+ * @link http://www.oomphinc.com
+ *
+ */
+
+class Braces_Custom_Taxonomy {
+	// Taxonomy
+	const TAX_SLUG     = 'braces';
+	const TAX_NAME     = 'Braces';
+	const TAX_SINGULAR = 'Braces';
+
+	function __construct() {
+		add_action( 'init', array( $this, 'action_init' ) );
+	}
+
+	/**
+	 * Register a Custom Taxonomy
+	 * @link https://codex.wordpress.org/Function_Reference/register_taxonomy
+	 */
+	function action_init() {
+		$labels = array(
+			'name'                       => _x( self::TAX_NAME, 'Taxonomy General Name', 'braces' ),
+			'singular_name'              => _x( self::TAX_SINGULAR, 'Taxonomy Singular Name', 'braces' ),
+			'menu_name'                  => __( self::TAX_SINGULAR, 'braces' ),
+			'all_items'                  => __( 'All '. self::TAX_NAME, 'braces' ),
```

Comment:
Similar comment as before, use `sprintf( _x( ... ) )` instead... and everywhere where a placeholder needs to be used as part of a translated string